test/fixtures/config-autoload/ts/object/postcss.config.ts

```diff
@@ -0,0 +1,18 @@
+import type { Config } from 'postcss-load-config';
+
+const config = function (api: Config) {
+  return {
+    parser: 'sugarss',
+    syntax: 'sugarss',
+    // FIXME: To confirm. Apparently there's no longer an available option `mode`?
```

Comment:
I need help with this one.

I am not sure whether this should be ignored or removed. This config key doesn't exist anymore.